### DISCUSSION TITLE
QA：常见问题查看这里

### DISCUSSION THREAD

**Discussion by oasis-cloud** [jdf2e/nutui-react#1324] [Q&A] [answered]
NutUI React 相关的一些使用问题或者限制

**Comment by oasis-cloud** [comment]
### 关于 Prebundle 与 Cache

表现1：使用组件后，微信小程序无内容，控制台警告找不到 template
表现2：控制台报错，找不到某某文件，文件名称中包含了 `prebundle`

可能的原因：开启了 Webpack 预加载或者 cache，Taro 框架下该模式会偶发丢失第三方组件库文件。
需要在 `config/index` 中将 NutUI 剔除掉并关闭 cache：
```js
module.exports = {
  // ...
  compiler: {
    type: 'webpack5',
    prebundle: {
      exclude: ['@nutui/nutui-react-taro', '@nutui/icons-react-taro']
    }
  },
  cache: {
    enable: false
  }
}
```

**Reply by CODEOrz** [reply]
这Taro的Prebundle感觉老是有问题，真烦。

**Comment by 5201314999** [comment]
希望能完善 form 表单复杂组件的使用demo

**Comment by xiaoyatong** [comment, marked as answer]
问题表现：
运行：pnpm run build:taro
错误：获取 taro 全局配置文件失败，不存在全局配置文件：C:\Users\XX\.taro-global-config\index.json

问题解决：可先查看以下issue：
https://github.com/NervJS/taro/issues/13963

**Comment by Nacy0624** [comment]
自定义宽度 无效

**Comment by eiinu** [comment]
### 出现 `[mini-css-extract-plugin] Conflicting order` 报错

> 环境：Taro 小程序

1、如果你的文件中出现了下面这种使用方式，在不同的页面中引入相同样式时的顺序不同，`mini-css-extract-plugin` 插件将认为这是错误的，因为样式的先后顺序可能会影响实际的表现效果。 
```js
// a.js
import 'button.css'
import 'cell.css'

// b.js
import 'cell.css'
import 'button.css'
```

2、NutUI React 使用 `babel-plugin-import` 插件实现样式的按需引入，如果你在不同的组件中使用了相同组件，但引入顺序不同，插件将会以不同的顺序引入相同的样式，导致警告出现。这不会影响产物的大小（会被 tree-shaking 掉）。

NutUI 各个组件的样式之间没有冲突，即**其顺序是无关紧要**的，所以可以通过以下设置来关闭这种错误提示：

```js
// config/index.js
export default {
  mini: {
    miniCssExtractPluginOption: {
      ignoreOrder: true
    }
  }
}
```

**Comment by mayu888** [comment]
input组件的placeholder字体颜色无法更改？

**Comment by oasis-cloud** [comment]
如何解决【@nutui/nutui-react-taro 使用 Iconfont 后，图标不展示】

在 app.ts 中添加 import '@nutui/icons-react-taro/dist/style_iconfont.css'

**Comment by linnaZ** [comment]
Toast 是否有配置支持同一时间下只弹一个
